### PR TITLE
Turbo cellar permissions

### PR DESCRIPTION
**Pull request by cbrit**

### CODE REVIEW

File: steward/src/cellars.rs [hunk 1]
```diff
@@ -33,18 +33,22 @@ pub const ORACLE3: (U256, &str) = (
     U256([5, 0, 0, 0]),
     "26cde3f5db92ea91c84c838e664fe42dec1b6747",
 );
-pub const ALLOWED_PRICE_ORACLES: [(U256, &str); 2] = [ORACLE1, ORACLE2];
+pub const ORACLE4: (U256, &str) = (
```
Comment (EricBolten) [inline steward/src/cellars.rs]: This is exactly the same as `ORACLE3`. We just needed to add it to the list.

File: steward/src/cellars.rs [hunk 2]
```diff
@@ -33,18 +33,22 @@ pub const ORACLE3: (U256, &str) = (
...
 pub const ALLOWED_CACHE_PRICE_ROUTER: [&str; 1] = [CELLAR_RYETH];
 
 // permissions
 
 pub const ALLOWED_V2_0_SETUP_ADAPTORS: [(&str, &str); 1] = [(CELLAR_RYUSD, ADAPTOR_CELLAR_V2)];
-pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 3] = [
-    (CELLAR_RYBTC, ADAPTOR_LEGACY_CELLAR_V1),
-    (CELLAR_RYBTC, ADAPTOR_COLLATERAL_F_TOKEN_V1),
-    (CELLAR_RYBTC, ADAPTOR_DEBT_F_TOKEN_V1),
+pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 0] = [];
```
Comment (EricBolten) [inline steward/src/cellars.rs:43]: Have we confirmed all the 2.2 adaptor add calls were made?

File: steward/src/cellars.rs [hunk 3]
```diff
@@ -70,13 +74,11 @@ pub const ALLOWED_V2_0_POSITIONS: [(&str, u32); 20] = [
     (CELLAR_RYUSD, 28),
     (CELLAR_RYUSD, 29),
 ];
-pub const ALLOWED_V2_2_CATALOGUE_POSITIONS: [(&str, u32); 4] = [
-    (CELLAR_RYBTC, 195),
-    (CELLAR_RYBTC, 196),
-    (CELLAR_RYBTC, 197),
-    (CELLAR_RYBTC, 198),
+pub const ALLOWED_V2_2_CATALOGUE_POSITIONS: [(&str, u32); 0] = [];
```
Comment (EricBolten) [inline steward/src/cellars.rs:73]: Same question here re: confirming calls have already been made.

**Review by EricBolten — COMMENTED** [review]
Some comments and questions.

File: steward/src/cellars.rs (hunk 2 above)
Comment (cbrit) [reply steward/src/cellars.rs:43]: No, looking for feedback from them in this PR

**Review by EricBolten — APPROVED** [review]
LGTM

**Pull request merged**